packages/web/config/chain-infos.ts

```diff
@@ -2901,6 +2901,13 @@ const chainInfos = IS_TESTNET
               coinGeckoId: "pool:staevmos",
               coinImageUrl: "/tokens/stevmos.svg",
             },
+            {
+              coinDenom: "stUMEE",
+              coinMinimalDenom: "stuumee",
+              coinDecimals: 6,
+              coinGeckoId: "pool:stuumee",
```

Comment:
Thanks for this. The last thing required is defining this price for 'pool:stuumee' in the prices.ts file. If you don't know how, I don't mind updating to your branch if you can let me know the pool id.

Reply:
Now that we know pool 1035, can we please define the price (named 'pool:stuumee') in the config/prices.ts file?